.github/workflows/release-metrics.yml

```diff
@@ -12,8 +12,7 @@ jobs:
           project_id: ${{ secrets.GCP_PROJECT_ID }}
           service_account_key: ${{ secrets.GCP_BQ_METRICS_KEY }}
           export_default_credentials: true
-      - run: 'echo ''${{ github.event }}'' |
```

Comment:
this translated to `echo 'Object' | jq -c '.release ...` - turns out there's actually a local file available with what I want located at `$GITHUB_EVENT_PATH`